lib/plugin.js

```diff
@@ -120,14 +120,32 @@ function setCookie(ctx, name, value, maxAge = <%= options.maxAge %>) {
 }
 
 // https://developers.google.com/optimize/devguides/experiments
-function googleOptimize({ experiment }) {
-  if (process.server || !window.ga || !experiment || !experiment.experimentID) {
+function googleOptimize(ctx) {
+  const { experiment } = ctx
+  if (process.server || !experiment || !experiment.experimentID) {
     return
   }
 
-  const exp = experiment.experimentID + '.' + experiment.$variantIndexes.join('-')
-
-  window.ga('set', 'exp', exp)
+  // Choose method to track experiment and variant
+  <% if (options.eventHandler === 'ga') { %>
+    // Use Google Analytics integration
+    // https://developers.google.com/optimize/devguides/experiments
+    const exp = experiment.experimentID + '.' + experiment.$variantIndexes.join('-')
+    if (!window.ga) return
+    window.ga('set', 'exp', exp)
+  <% } else if (options.eventHandler === 'gtm') { %>
+    // Use @nuxtjs/gtm-module
+    if (!ctx.$gtm) return
+    ctx.$gtm.push({
+      exp: exp
```

Comment:
you can use shorthands here

Reply:
Changed this and added const exp to the upper scope, which was a typo I found.